docs/reference/indices/analyze.asciidoc

```diff
@@ -56,11 +56,9 @@ the analyze API uses the <<analysis-standard-analyzer,standard analyzer>>.
 `analyzer`::
 +
 --
-(Optional, string or <<analysis-custom-analyzer,custom analyzer object>>)
-Analyzer used to analyze for the provided `text`.
-
-See <<analysis-analyzers>> for a list of built-in analyzers.
-You can also provide a <<analysis-custom-analyzer,custom analyzer>>.
+(Optional, string)
+The name of the analyzer that should be applied to the provided `text`. See <<analysis-analyzers>>
+for a list of built-in analyzers.
```

Comment:
Maybe we can mention that this also works for a custom analyzer that is defined in an index if "_analyze" is used on that index.

Reply:
I added a sentence to clarify that this can be any analyzer configured in the index, and not just a built-in analyzer.